src/storage/macros.rs

```diff
@@ -25,14 +27,26 @@
 /// define_intermediate!(1, Double -> i32, MyStorageType, |_: &Double, db: &DbHandle<MyStorageType>| {
 ///     db.get(MyInput) * 2
 /// });
+///
+/// // It is also possible to signal that the value always changes with the assume_changed keyword:
```

Comment:
```suggestion
/// // It is also possible to signal that the value always changes with the assume_changed keyword.
/// // Doing so let's us avoid expensive `Eq` checks on large values which are expected to change whenever their inputs do anyway:
```
Adding some motivation context to this comment to make it more clear when users may want to do this.